app/api/localStorage/index.js

```diff
@@ -195,10 +196,48 @@ export default class LocalStorageApi {
     } catch (error) {} // eslint-disable-line
   });
 
+  // ========== Show/hide Balance ========== //
+
+  getHideBalance = (): Promise<boolean> => new Promise((resolve, reject) => {
+    try {
+      const hideBalance = localStorage.getItem(storageKeys.HIDE_BALANCE);
+      if (!hideBalance) return resolve(false);
```

Comment:
I may be wrong, but I think `resolve` and `reject` shouldn’t return. I would add an else below this as well.

Reply:
Right. `return` should be after resolve (it's just used to stop execution). With an `else` it's not required.
Note that some parts of the rest of the code also use that construction.

Reply:
Oh, interesting 🤔 In other programming languages with other libraries it would generate a problem. I guess this library detects it and cleans it up?